.storybook/storybook-data.js

```diff
@@ -1,4 +1,71 @@
 module.exports = [
+	{
+		id: 'components-dashboardcoresitealerts--notification-cta',
+		kind: 'Global',
+		name: 'DashboardCoreSiteAlerts1',
+		story: 'DashboardCoreSiteAlerts1',
+		parameters: {
+			fileName: './assets/js/components/legacy-notifications/DashboardCoreSiteAlerts.stories.js',
+			options: {
+				hierarchyRootSeparator: '|',
+				hierarchySeparator: {},
+				readySelector: '.googlesitekit-publisher-win',
+			},
+		},
+	},
+	{
+		id: 'components-dashboardcoresitealerts--no-notifications',
+		kind: 'Global',
+		name: 'DashboardCoreSiteAlerts2',
+		story: 'DashboardCoreSiteAlerts2',
+		parameters: {
+			fileName: './assets/js/components/legacy-notifications/DashboardCoreSiteAlerts.stories.js',
+			options: {
+				hierarchyRootSeparator: '|',
+				hierarchySeparator: {},
+			},
+		},
+	},
+	{
+		id: 'components-dashboardcoresitealerts--notification-cta-with-survey',
+		kind: 'Global',
+		name: 'DashboardCoreSiteAlerts3',
+		story: 'DashboardCoreSiteAlerts3',
+		parameters: {
+			fileName: './assets/js/components/legacy-notifications/DashboardCoreSiteAlerts.stories.js',
+			options: {
+				hierarchyRootSeparator: '|',
+				hierarchySeparator: {},
+			},
+		},
+	},
```

Comment:
I get why these empty stories are here, but they're a bit confusing—it might make someone wonder what's up with them in the future. Maybe some comments explaining them would help? Especially as the story names aren't very helpful with the Storybook V6 naming.

Reply:
Updated both stories and backstop in https://github.com/google/site-kit-wp/pull/3603/commits/fe626911e4f20724fb67b1fc801675bc55118508